FAQ.md

```diff
@@ -1,7 +1,129 @@
+This also explains why codes are single use. Once a code is used ‘successfully’ (sender and recipient connect) or ‘unsuccessfully’ (a wrong code combination is guessed), the code is cleared for new usage. If the code would stay ‘alive’ for other recipients, it would lose the security of being hard to guess.
+
+Can I send multiple files at the same time?  
+If you would like to send multiple files at the same time, you can compress them, for example, as a single ZIP file. But you cannot upload an unzipped folder or select multiple files to upload at once. You will need to send them separately and provide the receiver with a new code each time. 
+
+
+**At what point can I retract a file that I am sending?** 
+As a sender you can first cancel a transfer before sharing the code by pressing the Cancel button. After sharing the code you can still press Cancel if the other party has not yet started downloading the file. If you press Cancel while the transfer is ongoing, it is possible for the receiver to already have a portion of the file.
+
+**Can I use Destiny to send/receive files to/from other Magic Wormhole applications?**  
+Yes. Make sure to configure the other Magic Wormhole application to use the same environment settings set in Destiny. These are currently:
+
+	Mailbox URL: wss://mailbox.w.leastauthority.com/v1
+	Transit relay URL: tcp://relay.w.leastauthority.com:4001
+	AppID: lothar.com/wormhole/text-or-file-xfer
+
+**What are the known issues or limitations of Destiny?** 
+Error messages: we are not always able to determine the exact reason for an error occurring and therefore not able to provide the exact troubleshooting steps. For example, it is difficult to know whether an error occurred due to a network interruption or due to a cancellation.
+
+Cancellation: Canceling a transfer may lead to the application getting stuck during the transfer. And as previously mentioned, if the sender presses Cancel while the transfer is ongoing, it is possible for the receiver to already have a portion of the file.
+
+Inability to change wormhole settings: a user is not able to change certain wormhole-william settings such as the mailbox server, relay server, and/or the passphrase component length. 
+
+No iOS support: we currently do not offer an iOS version of the application. 
+
+Unable to set default file destination for Android: a user is not able to change the default directory in Android, so files are all automatically saved to the default Downloads folder. 
+
+Interoperability challenges: While we are striving to ensure interoperability between Destiny and other applications using the Magic Wormhole protocol, this has not fully been tested. As such, we cannot at this stage guarantee full interoperability.  
+
+We will work on addressing these issues in V2 of the application. We welcome external contributions to the code and improvements. In the meantime, please report any other bugs and suggestions to destiny@leastauthority.com. 
+
+
+**Contact**
```

Comment:
We need to have headline: ## CONTACT
as I user in url in app: https://github.com/LeastAuthority/destiny/blob/main/FAQ.md#contact